docker/blackduck-alert-db/alertdb-pg-migrate.sh

```diff
@@ -179,8 +190,8 @@ backupPostgresDataDirectory="${runDirectory}/backup.${epochTime}"
 
 pgHbaConfFileName="pg_hba.conf"
 
-postgresUser=""
-postgresPassword=""
+postgresUser="${POSTGRES_ADMIN_USER:-${POSTGRES_USER}}"
+postgresPassword="${POSTGRES_ADMIN_PASSWORD:-${POSTGRES_PASSWORD}}"
 
```

Comment:
The PG user needs to be POSTGRES_ADMIN_USER for Centos upgrade, and POSTGRES_USER for Alpine upgrades. This is for issue 1